### DISCUSSION TITLE
One Click Install option?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17217] [Archive]
I backed up a project last year following DOC instructions. (.sql database dump, /upload folder, and .env file)
I am not a backend or engineer, so I relied on a one-click solution for digital ocean droplets, which is no longer available.
My PostgreSQL database is set up and running as a managed instance. in DO.
Can anyone point me to a good tutorial on HOW TO SETUP DIRECTUS on the DigitalOcean App engine? With Spaces as CDN and Managed database environment? All the DOCS on that subject are now removed.

**Comment by rijkvanzanten** [comment]
Use the Docker Image 👍🏻 From a quick Google search, it looks like DO has loads of in-depth articles on how to run Docker images on App Platform

**Comment by ghost** [comment]
I am facing this error when connecting to my managed pg database. Tried many different ways to add the CA cert as a variable but I don't think that's possible in D.O. I am using directs/directus docker container. As you guys can guess I really don't have much DevOps expertise, so if you can give me any direction or point to any tutorial that includes the steps to get it running, it will be much appreciated. 

<img width="896" alt="Screenshot 2023-01-20 at 13 23 07" src="https://user-images.githubusercontent.com/122748607/213751093-e852f2e9-ef5f-42fe-84c4-befde7ad65a0.png">

**Reply by ghost** [reply]
I also tried with DB_SSL__REJECT_UNAUTHORIZED="false" added to env variables but still same error.

**Reply by rijkvanzanten** [reply]
>  As you guys can guess I really don't have much DevOps expertise, so if you can give me any direction or point to any tutorial that includes the steps to get it running, it will be much appreciated.

This is why we built Directus Cloud! DevOps is difficult, keeping things alive and making it scalable is even harder.

**Reply by ghost** [reply]
Yeah, I want to use it, but I could not find a way to import my current database with all the collections and data.

**Reply by rijkvanzanten** [reply]
Thanks for the feedback! We're currently working on enabling that for cloud (not a small feat from a security perspective!) so hopefully this is going to be a thing of the past soon :)

**Comment by ghost** [comment]
Worked! I used NODE_TLS_REJECT_UNAUTHORIZED="0" instead of DB_SSL__REJECT_UNAUTHORIZED="false"

**Reply by rijkvanzanten** [reply]
Careful though! That effectively opens you up to man in the middle attacks, as it no longer verifies the CA of the certificate.

**Reply by paescuj** [reply]
Was about to say the same thing! 🦭 

Most secure way is to get the CA certificate (see here for DigitalOcean: https://docs.digitalocean.com/products/databases/postgresql/how-to/connect/) and actually use it to verify the SSL connection:
```
DB_SSL__CA_FILE=path/to/ca.crt
```

Alternatively, instead of using `NODE_TLS_REJECT_UNAUTHORIZED=0` (which applies to the whole Node.js scope, incl. HTTP requests, sending mails, ...) it would still be a tad better to use `DB_SSL__REJECT_UNAUTHORIZED=false` (which only applies to the database connection).
This is expected to work! I guess in the case of https://github.com/directus/directus/discussions/17217#discussioncomment-4739133 the problem was that `DB_SSL=true` was defined at the same time, while only one of the two may be set (see also https://github.com/directus/directus/discussions/2997#discussioncomment-258584).
